apps/happychat/src/index.ejs

```diff
@@ -22,19 +22,12 @@
 	<body>
 		<div id="wpcom" class="container"></div>
 		<script>
-			const env = new URLSearchParams( window.location.search ).get( 'env' ) || 'dev';
-			const happychat_url =
-				env === 'dev'
-					? 'https://happychat-io-staging.go-vip.co/customer'
-					: 'https://happychat.io/customer';
-			const env_id = env === 'dev' ? 'development' : 'production';
-
 			window.configData = {
-				env_id,
+				env_id: 'development',
```

Comment:
Oooh I hope this one doesn't complicate things. 

I searched the code and Happychat doesn't seem to depend on this, we're probably OK to delete it. Would appreciate any insight on this one @klimeryk 🙏🏼 .

Reply:
Removed and tested in dev / prod both seem to start correctly in widgets.wp.com 👍